src/config.js

```diff
@@ -33,8 +33,16 @@ module.exports = {
     hostname: process.env.ANALYTICS_HOSTNAME || "",
   },
 
+  postgres: {
+    host : process.env.POSTGRES_HOST,
+    user : process.env.POSTGRES_USER,
+    password : process.env.POSTGRES_PASSWORD,
+    database : process.env.POSTGRES_DATABASE || "analytics-reporter",
+  },
+
   static: {
-    path: '../analytics.usa.gov/'
+    path: '../analytics.usa.gov/',
   },
 
+  timezone: process.env.ANALYTICS_TIMEZONE || "US/Eastern",
```

Comment:
I had assumed the timezone effect would be global, whether or not they used the database writing feature. I'd actually prefer to make the change all at once, for exactly the reason you suggested. I don't want to hold up the PR if it's a huge deal, but I also think it's weird to potentially have different time zone behavior in the generated data depending on whether or not postgres syncing was enabled at publish-time.